### PR TITLE
Simple IUser interface with example user type usage

### PR DESCRIPTION
**Pull request by nicki419**

### CODE REVIEW

File: IUser.cs
Comment (nicki419) [inline IUser.cs:1]: Should be its own file.

File: IUser.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public interface IUser
+{
+	int UserID { get; set; }
+	string Name { get; set; }
+	string Email { get; set; }
+	string Address { get; set; }
+}
+public class Admin : IUser
+{
```
Comment (nicki419) [inline IUser.cs:11]: Should be its own file.

File: IUser.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	public int UserID { get; set; }
+	public string Name { get; set; }
+	public string Email { get; set; }
+	public string Address { get; set; }
+
+	public bool ManageUsers { get; set; }
+}
+
+public class Employee : IUser
+{
```
Comment (nicki419) [inline IUser.cs:21]: Should be its own file.

File: IUser.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	string Address { get; set; }
+}
+public class Admin : IUser
+{
+	public int UserID { get; set; }
+	public string Name { get; set; }
+	public string Email { get; set; }
+	public string Address { get; set; }
+
+	public bool ManageUsers { get; set; }
```
Comment (nicki419) [inline IUser.cs:17]: This is a parameter, but it is named like a method. What does it do? Why is the return type bool?

File: IUser.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+public class Employee : IUser
+{
+	public int UserID { get; set; }
+	public string Name { get; set; }
+	public string Email { get; set; }
+	public string Address { get; set; }
+
```
Comment (nicki419) [inline IUser.cs:26]: Unnecessary whitespace

File: IUser.cs [hunk 5]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+public interface IUser
+{
+	int UserID { get; set; }
+	string Name { get; set; }
+	string Email { get; set; }
+	string Address { get; set; }
```
Comment (nicki419) [inline IUser.cs:8]: Doubt thats necessary for our usecase. We manage tables, not employees in general.

File: IUser.cs [hunk 6]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+public interface IUser
+{
```
Comment (nicki419) [inline IUser.cs:4]: There should be methods responsible for retreiving a user from the database. Probably as a constructor. 
Should be implemented in the interface, for the separate users, you can simply throw a NotImplemented exception.

**Review by nicki419 — COMMENTED** [review]
There should be a maintenance / cleaner user type.

File: IUser.cs (hunk 3 above)
Comment (Nikozzh) [reply IUser.cs:17]: Suggested by copilot <3
